src/main/java/server/Server.java

```diff
@@ -0,0 +1,132 @@
+package server;
+
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Server {
+
+    private static final int PORT = 8080;
+    private static final String EXIT_COMMAND = "-exit";
+    private static final String FILE_COMMAND = "-file";
+
+    private ServerSocket serverSocket;
+    private List<ClientHandler> clients;
+
+    public Server() {
+        clients = new ArrayList<>();
+    }
+
+    public void start() {
+        try {
+            serverSocket = new ServerSocket(PORT);
+            System.out.println("Server started on port " + PORT);
+
+            while (true) {
+                Socket clientSocket = serverSocket.accept();
+                ClientHandler clientHandler = new ClientHandler(clientSocket);
+                clients.add(clientHandler);
+                clientHandler.start();
+                broadcastMessage("[SERVER] " + clientHandler.getClientName() + " connected successfully.");
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void stop() {
+        try {
+            serverSocket.close();
+            for (ClientHandler client : clients) {
+                client.disconnect();
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void broadcastMessage(String message) {
+        for (ClientHandler client : clients) {
+            client.sendMessage(message);
+        }
```

Comment:
Only one client should receive a message, not everyone. If client 1 closed its connection, why would client 2 receive a message about that event? This info is only for client 1.